self-reflection-1-myerskevin14.qmd

```diff
@@ -0,0 +1,74 @@
+
+2.  Once you've verified that you've correctly cloned the assignment repository, [create a new Quarto document](https://quarto.org/docs/get-started/authoring/rstudio.html). Name this file `self-reflection-1-xxx.qmd` and give it a title (like M Williamson Self-Reflection 1). Make sure that you select the `html` output option (Quarto can do a lot of cool things, but the html format is the least-likely to cause you additional headaches). We'll be using Quarto throughout the course so it's worth checking out the other tutorials in the getting started section.
+
+3.  Copy the questions below into your document and [change the color of their text](https://meghan.rbind.io/blog/quarto-slides/).
+
+4.  Save the changes and make your first commit!
+
+5.  Answer the questions making sure **save and commit at least 4 more times** (having 5 commits is part of the assignment).
+
+6.  Render the document to html (you should now have at least 3 files in the repository: Readme.md, self-reflection-1-xx.qmd, and self-reflection-1-xx.html). Commit these changes and push them to the repository on GitHub. You should see the files there when you go to github.com.
+
+## Questions
+
+**About you**
+
+1.  Please tell me a little bit about your research interests and how you imagine integrating spatial data in your research.
+
+    I am currently working with movement data from satellite and cellular transmitters affixed to juvenile golden eagles. My research pertains to the period of time when the young bird has left its nest and is being taught by their parents how to fly and hunt. I plan to integrate spatial data into my research by mapping where the young are flying and determining the timing of the onset of independence.  
+
+2.  How do you learn best, and what do you feel is your biggest strength as a student?
+
+    I learn best by following along in class with codes and independent work. I think that my biggest strength as a student is problem solving skills. I feel confident that when I get stuck or confused, I can look up my questions or ask the instructor for help.
+
+3.  If you want to, you can set a goal for this semester for something you want to work on in the context of this course (outside of the topic matter itself) -- for example, using StackOverflow to get answers to analysis questions or using git to track progress on homeworks. If you choose to set this type of goal, what can you do to help support yourself in working toward that goal? How can I help you to meet this goal?
+
+    I would like to be able to use StackOverflow efficiently as well as becoming more familiar/confident using git and github. When problems arise, starting with StackOverflow/git instead of just googling or using chatgpt for help will make this goal more achievable. Also participating in class and asking for help in the moment instead of waiting until after class will also facilitate accomplishing this goal. 
```

Comment:
As we progress in the class and you have some homeworks under your belt, feel free to reassess yourself and let me know how it's going! I'm sure there will be times you will want to consult StackOverflow once we start working with data.